fiftyone/operators/executor.py

```diff
@@ -430,20 +430,25 @@ def results(self):
         return self.request_params.get("results", {})
 
     @property
-    def secrets(self) -> dict:
-        """The dict of secrets available to the operation (if any)."""
-        return self._secrets
+    def secrets(self) -> list[str]:
```

Comment:
We've already released and documented `ctx.secrets` as a dict. Why do we need a breaking change here?

Reply:
- https://docs.voxel51.com/plugins/using_plugins.html#plugin-secrets
- https://docs.voxel51.com/teams/secrets.html#using-secrets
- https://docs.voxel51.com/plugins/developing_plugins.html#accessing-secrets

Reply:
To avoid confusion on why a secret is None or potential errors raised from using ctx.secrets[key] and encourage users to use ctx.secret(key)